src/js/media-gesture-receiver.js

```diff
@@ -54,9 +50,19 @@ class MediaGestureReceiver extends window.HTMLElement {
       this._pointerType = e.pointerType;
     };
 
-    this.addEventListener('pointerdown', pointerDownHandler);
+    window.addEventListener('pointerdown', pointerDownHandler);
```

Comment:
issue(blocker): let's move the adding of the event handler to `connectedCallback` and add a corresponding remove to `disconnectedCallback` (can add a private method for `pointerDownHandler`)